src/ini/config.c

```diff
@@ -0,0 +1,81 @@
+#include "ini.h"
+
+typedef struct /*Can be changed in the future*/
+{
+    const char* orchestratorIP;
+    const char* ochestratorPort;
+} configuration_node;
```

Comment:
The [code style from systemd](https://github.com/systemd/systemd/blob/main/docs/CODING_STYLE.md#code-organization-and-semantics) suggests naming structs in `PascalCase`
```suggestion
} NodeConfiguration;
```

Reply:
And the variables in snake_case